src/main/java/at/klausbetz/provider/AppleIdentityProvider.java

```diff
@@ -73,7 +73,7 @@ public Object callback(RealmModel realm, AuthenticationCallback callback, EventB
 
     @Override
     protected String getDefaultScopes() {
-        return "openid name email";
+        return "name%20email";
```

Comment:
Hi @klausbetz,
Could you confirm that scope sent to Apple in HTTP is `name%20email`?   I see in OIDCIdentityProvider is:
```
        String defaultScope = config.getDefaultScope();

        if (!defaultScope.contains(SCOPE_OPENID)) {
            config.setDefaultScope((SCOPE_OPENID + " " + defaultScope).trim());
        }
```
I'm using different provider, but implementation is similar and see in HTTP: `scope=openid%20name%20email`

Kindly regards,